components/page-header/private/record-home/index.jsx

```diff
@@ -40,17 +40,28 @@ const RecordHome = (props) => (
 			<div className="slds-col slds-has-flexi-truncate">
 				<MediaObject
 					body={
-						<div>
-							{props.label}
-							{props.title}
-							{props.info}
+						<div className="slds-page-header__name">
+							<div className="slds-page-header__name-title">
+								<h1>
+									{props.label}
+									{props.title}
+								</h1>
+							</div>
 						</div>
 					}
 					className="slds-no-space slds-grow"
 					figure={props.icon}
 				/>
 			</div>
-			<div className="slds-col slds-no-flex slds-grid slds-align-top">
+			<div className="slds-col slds-no-flex slds-grid slds-align-top slds-p-bottom_xx-small">
+				{props.navRight}
+			</div>
```

Comment:
I'm not 100% on this (breaking) change from `contentRight` to `navRight`. Without this, on a `recordHome` variant the `contentRight` is used in much the same way `navRight` is used elsewhere. This change standardizes it to avoid confusion.